test/trento_web/controllers/v1/cluster_controller_test.exs

```diff
@@ -458,6 +458,54 @@ defmodule TrentoWeb.V1.ClusterControllerTest do
         |> assert_schema("Forbidden", api_spec)
       end
     end
+
+    test "should accept cluster_host_start operation for a valid host and the user is allowed" do
```

Comment:
nitpick:
for consistency, maybe we could have a "matrix" kind of test to reuse the code, like we do above with pacemaker scenarios. even add these scenarios to that test.
We would cover much more things, like amqp errors, etc.
anyway, just a personal opinion and how @nelsonkopliku and I did the testing for the rest of operations